libraries/nrpe_installation_archive.rb

```diff
@@ -95,7 +95,7 @@ def nagios_plugins
       # @return [String]
       # @api private
       def nrpe_program
-        options.fetch(:program, ::File.join(nrpe_base, 'bin', 'nrpe'))
+        options.fetch(:program, @program)
```

Comment:
Where are we storing that instance variable? The second parameter is the default value.